frontend/public/components/catalog-items.jsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as PropTypes from 'prop-types';
+import * as classNames from 'classnames';
+
+import {CatalogTileView} from 'patternfly-react-extensions/dist/esm/components/CatalogTileView';
+import {CatalogTile} from 'patternfly-react-extensions/dist/esm/components/CatalogTile';
+import {normalizeIconClass} from './catalog-item-icon';
+
+const CatalogItem = ({item}) => {
+  return <CatalogTile
+    id={item.obj.metadata.uid}
+    key={item.obj.metadata.uid}
+    title={item.tileName}
+    iconImg={item.tileImgUrl}
+    iconClass={item.tileIconClass ? `icon ${normalizeIconClass(item.tileIconClass)}` : null}
+    vendor={item.tileProvider ? `Provided by ${item.tileProvider}` : null}
+    description={item.tileDescription}
+    onClick={item.callback}
```

Comment:
It's better to accessibility to do something like

```jsx
<a href={item.href}>
  <CatalogTile ... />
</a>
```

But OK for now.

Reply:
fixed